### ISSUE TITLE
Argdown-Cli Test failing on Windows

### ISSUE THREAD

**Issue by cwkit** [christianvoigt/argdown#68]
Error Message:

```
 1) argdown-cli
       can load glob input:
     Uncaught AssertionError: value: expected 'D:\\Code-Projekte\\argdown\\packages\\argdown-cli\\test\\json\\test.json' to exist
      at Proxy.<anonymous> (D:\Code-Projekte\argdown\packages\argdown-cli\node_modules\chai-fs\lib\assertions\file.js:21:53)
      at Proxy.methodWrapper (D:\Code-Projekte\argdown\packages\argdown-cli\node_modules\chai\lib\chai\utils\addMethod.js:57:25)
      at execPromise (D:\Code-Projekte\argdown\packages\argdown-cli\test\argdown-cli.spec.ts:230:50)
      at require.exec (D:\Code-Projekte\argdown\packages\argdown-cli\test\argdown-cli.spec.ts:27:7)
      at ChildProcess.exithandler (child_process.js:267:7)
      at emitTwo (events.js:126:13)
      at ChildProcess.emit (events.js:214:7)
      at maybeClose (internal/child_process.js:925:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
```

**Comment by cwkit** [comment]
```
Error also occurs with latest version

```
1) argdown-cli
       can load glob input:
     Uncaught AssertionError: value: expected 'D:\\Code-Projekte\\argdown\\packages\\argdown-cli\\test\\json\\test.json' to exist
      at Proxy.<anonymous> (D:\Code-Projekte\argdown\packages\argdown-cli\node_modules\chai-fs\lib\assertions\file.js:21:53)
      at Proxy.methodWrapper (D:\Code-Projekte\argdown\packages\argdown-cli\node_modules\chai\lib\chai\utils\addMethod.js:57:25)
      at execPromise (D:\Code-Projekte\argdown\packages\argdown-cli\test\argdown-cli.spec.ts:230:50)
      at require.exec (D:\Code-Projekte\argdown\packages\argdown-cli\test\argdown-cli.spec.ts:27:7)
      at ChildProcess.exithandler (child_process.js:267:7)
      at emitTwo (events.js:126:13)
      at ChildProcess.emit (events.js:214:7)
      at maybeClose (internal/child_process.js:925:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
```

**Comment by christianvoigt** [comment]
fixed in 1.0.0 (no auto-closing because of rebased monorepo branch).
